src/index.ts

```diff
@@ -83,6 +83,15 @@ export default class NewRelicLambdaLayerPlugin {
       return;
     }
 
+    const { exclude = [] } = this.config;
+    const { include } = this.config;
```

Comment:
I'd suggest that include and exclude should have identical structures to reduce any future confusion.  So 
``` const { include = [], exclude = [] } = this.config ```

Reply:
good point aligned the functionality

Reply:
And then that could be carried throughout the file where that's used.

Reply:
they are different methods which means they would need to be passed through as arguments

but I've noticed that include was missing in subscription (added now)

Reply:
I think you meant defaulting to an empty array. Yes, I've corrected that as part of fix so it follows the same approach.

Reply:
Yes! Apologies for the shorthand there.

Reply:
cool I've extracted to common function and used the shorthand 

looks better 😄